backend/db/utils.py

```diff
@@ -119,6 +121,7 @@ def refresh_any_dependent_tables(con: Connection, independent_tables: List[str]
     derived_tables: List[str] = get_dependent_tables_queue(independent_tables)
     if not derived_tables:
         print(f'No derived tables found for: {", ".join(independent_tables)}')
+        return
```

Comment:
## Unrelated update: return early when no derived tables found
The function `refresh_any_dependent_tables()` checks a given table (e.g. `code_sets`) and finds any tables dependent on it (e.g. `all_csets`), then proceeds to call `refresh_derived_tables()`. If it didn't find any dependent table, it would print a message like "no tables now", but it would continue onto the `refresh_derived_tables()` function. `refresh_derived_tables()` doesn't do anything if it is not passed any tables, so this was harmless, but I decided it's much better to do a return and not call it.